zitilib/characterization/ziti/operation.go

```diff
@@ -68,8 +69,8 @@ func (f *operationFactory) Build(m *model.Model) error {
 
 	c := make(chan struct{})
 	m.Operation = model.OperatingBinders{
-		func(m *model.Model) model.OperatingStage { return operation.Mesh(c) },
-		func(m *model.Model) model.OperatingStage { return operation.Metrics(c) },
+		func(m *model.Model) model.OperatingStage { return __operation.Mesh(c) },
+		func(m *model.Model) model.OperatingStage { return __operation.Metrics(c) },
```

Comment:
Have you seen this used in other repos? Never thought of doing it.

Reply:
Goland did it during refactoring.